### PR TITLE
#169256284 implement profile update modal

### PR DESCRIPTION
**Pull request by placideirandora**
#### What does this PR do?
It implements the functionality of updating a user profile with a modal.

#### What to check
1. If you update the profile with a modal.
2. If the image is previewed prior to the update.

#### How should this be manually tested?
1. Clone the branch.
2. Do **npm install**.
3. Do **npm start**.
4. Login and update your profile

#### Screenshots
Screen Shot 2019-10-29 at 15 29 34
Screen Shot 2019-10-29 at 15 30 11

### CODE REVIEW

**Review by raymond42 — COMMENTED** [review]
Looks good to me

**Review by malaba6 — COMMENTED** [review]
The profile page looks better than it was before and the image is preview is working. It looks good to me...

**Comment by harerakalex** [comment]
Looks good so far. But how about the responsiveness.

**Comment by placideirandora** [comment]
> Looks good so far. But how about the responsiveness.

I worked on it.

**Review by harerakalex — APPROVED** [review]
Approve

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
```
Comment (malaba6) [inline .vscode/launch.json:3]: What is this VSCode file doing here?

File: src/feature/profile/view_profile/__tests__/ViewProfileComponent.test.js [hunk 2]
```diff
@@ -1,17 +1,40 @@
...
+
+  it('should render the view profile component', () => {
+    expect(wrapper.length).toEqual(1);
+  });
   it('should render the view profile component', () => {
-    expect(Wrapper.exists()).toBe(true);
+    expect(wrapper.length).toBe(1);
+  });
+  it('should click on the model', () => {
+    button.simulate('click');
```
Comment (Jkadhuwa) [inline src/feature/profile/view_profile/__tests__/ViewProfileComponent.test.js:38]: You should expect something to happen like action being fired.

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (Jkadhuwa) [inline .vscode/launch.json:1]: What's the use of this file?

**Review by Jkadhuwa — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 3 above)
Comment (placideirandora) [reply .vscode/launch.json:1]: It is meant to configure VSCode Debugging.

File: src/feature/profile/view_profile/__tests__/ViewProfileComponent.test.js (hunk 2 above)
Comment (placideirandora) [reply src/feature/profile/view_profile/__tests__/ViewProfileComponent.test.js:38]: You're right! Feedback implemented.

**Review by malaba6 — APPROVED** [review]

**Review by malaba6 — APPROVED** [review]

**Review by Jkadhuwa — APPROVED** [review]

**Pull request merged**